### DISCUSSION TITLE
Good places too learn Node Js

### DISCUSSION THREAD

**Discussion by ghost** [community/community#68162] [Programming Help]
### Body

Hi,

I am a beginner web developer now moving on to backend development can you recommend any simplified tutorials for Node.js?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by seyLu** [comment]
#### Hi @animeReview,

[Brad Traversy's Node.js crash course](https://www.youtube.com/watch?v=fBNz5xF-Kx4) is a good place to start -- make sure you learn the prerequisites mentioned in the vid first.

But if you have a time to spare for a more comprehensive course, then I recommend taking [Full Stack Open](https://fullstackopen.com/en/about)

**Comment by varmaexe** [comment]
Hi @animeReview ,

[Procademy's](https://youtube.com/playlist?list=PL1BztTYDF-QPdTvgsjf8HOwO4ZVl_LhxS&feature=shared) Playlist is good to learn. He covered all the topics in this playlist. 

All the best. 
Also I want to learn NodeJs too. feel free to message me

**Comment by devinscheung** [comment]
Hi @animeReview 

1. [Codecademy's](https://www.codecademy.com) has interactive Node.js Lessons.
2. [Udemy](https://www.udemy.com) courses usually provide step to step project based tutorials.
3. [w3schools](https://www.w3schools.com/nodejs/) is a good place for beginner.
4. The official [Node.js](https://nodejs.dev/en/learn/) website has a guide section which is really helpful.

Good luck :)

**Comment by jsdevmaster** [comment]
I think [w3schools.com](https://www.w3schools.com/) is good for beginner
